src/cpp/src/tokenizer.cpp

```diff
@@ -59,6 +59,19 @@ constexpr char bos_token_key_name[] = "bos_token";
 constexpr char eos_token_key_name[] = "eos_token";
 constexpr char pad_token_key_name[] = "pad_token";
 
+ov::Core core_with_extension() {
+    ov::Core core;
+    const char* ov_tokenizer_path = getenv(ScopedVar::ENVIRONMENT_VARIABLE_NAME);
+    OPENVINO_ASSERT(ov_tokenizer_path, "openvino_tokenizers path is not set");
+    core.add_extension(ov_tokenizer_path);
```

Comment:
I still think that default singleton from utils.hpp can include tokenizer extension to avoid duplication of logic of multiples Cores creation.